.fixtures.yml

```diff
@@ -1,13 +1,13 @@
 fixtures:
   repositories:
     "stdlib":
-      "repo": "git://github.com/puppetlabs/puppetlabs-stdlib.git"
-      "ref": "master"
+      "repo": "https://github.com/puppetlabs/puppetlabs-stdlib.git"
+      "ref": "main"
     "concat":
-      "repo": "git://github.com/puppetlabs/puppetlabs-concat.git"
-      "ref": "master"
+      "repo": "https://github.com/puppetlabs/puppetlabs-concat.git"
+      "ref": "main"
     "hocon":
-      "repo": "git://github.com/puppetlabs/puppetlabs-hocon.git"
+      "repo": "https://github.com/voxpupuli/puppet-hocon.git"
       "ref": "master"
   symlinks:
```

Comment:
you can remove the symlink option. it's not required anymore

Reply:
:+1: awesome, thanks!